crates/re_viewer_context/src/viewer_context.rs

```diff
@@ -150,3 +150,61 @@ pub struct RecordingConfig {
     /// The current time of the time panel, how fast it is moving, etc.
     pub time_ctrl: RwLock<TimeControl>,
 }
+
+#[cfg(test)]
+mod text {
+    use re_log_types::{StoreId, StoreKind};
+
+    use super::*;
+    use crate::command_channel;
+
+    /// This test is meant to ensure that a `ViewerContext` is "easy" to crate, e.g. for test
+    /// harnesses.
+    #[test]
+    fn test_viewer_context_default() {
```

Comment:
given that we have `TestContext` is this needed?

Reply:
Right. Gone.